cdap-stream-clients/java/README.rst

```diff
@@ -60,7 +65,7 @@ Example::
 
 Creating a Stream
 -----------------
-Create a new Stream with the *stream id* "streamName"::
+Create a new Stream with the *stream-id* "streamName"::
```

Comment:
Hmm not sure what the formatting should be... what is the convention elsewhere? What is italic used for in other places?